### PR TITLE
Add producerCount, WaterInjectorCount, gasInjectorCount to Surf PROSP import

### PR DESCRIPTION
**Pull request by petterwildhagen**

### CODE REVIEW

**Review by ahmedoabdi — CHANGES_REQUESTED** [review]
Check comment on gas- and water-injector

File: backend/api/Services/ImportProspService.cs [hunk 1]
```diff
@@ -95,6 +95,9 @@ private void ImportSurf(List<Cell> cellData, Guid sourceCaseId, Guid projectId)
             var artificialLift = MapArtificialLift(artificialLiftInt);
             var riserCount = ReadIntValue(cellData, "K36");
             var templateCount = ReadIntValue(cellData, "K32");
+            var producerCount = ReadIntValue(cellData, "E38");
+            var gasInjectorCount = ReadIntValue(cellData, "E39");
+            var waterInjectorCount = ReadIntValue(cellData, "E40");
```
[the remark this answers is not in the document]
Comment (petterwildhagen) [reply backend/api/Services/ImportProspService.cs]: Good catch. Fixed

[the remark this answers is not in the document]
Comment (ahmedoabdi) [reply backend/api/Services/ImportProspService.cs]: Tested, works well. Nice job 👍🏿

**Review by ahmedoabdi — APPROVED** [review]
LGTM!

**Pull request merged**